cli/program.js

```diff
@@ -169,13 +208,17 @@ const buildCommands = ({ program, commands = {}, envBuilder, env, loadCommand })
           ...program.opts(),
           ...cmdOptions,
           commandName: cmdName,
+          blueprints: envBuilder.getBlueprintsOption(),
         };
         if (options.installPath) {
           // eslint-disable-next-line no-console
           console.log(`Using jhipster at ${path.dirname(__dirname)}`);
           return undefined;
         }
 
+        printLogo();
+        printBlueprintLogo();
```

Comment:
Right now, since logo is printed at the generator, its up to the blueprint. See:
https://github.com/jhipster/jhipster-kotlin/blob/690b0621ebfdeb27533f76d4eba34b0a6b2e68c3/generators/app/index.js#L36-L39

Current approach, `jhipster --blueprints kotlin` will not print JHipster logo at all.
And `jhipster entity foo` doesn't print either, since the generator doesn't implements displayLogo.

So change actually forces every `jhipster` command to print JHipster logo and allows `khipster` command to omit it.
Since there is no official definition about this, IMO we show keep the decision to blueprints developers to customize it in its own cli.